test/giphy_sdk_test.dart

```diff
@@ -18,6 +18,6 @@ void main() {
   });
 
   test('getPlatformVersion', () async {
-    expect(await GiphySdk.connectToGiphy(apiKey: ""), true);
+    expect(await GiphySdk.openGiphySelection(apiKey: ""), true);
```

Comment:
I guess this test was changed by automated refactoring? Maybe add actual tests here if possible

Reply:
Would leave as is